source/administration/bucket-replication.rst

```diff
@@ -176,45 +176,29 @@ application of object expiration.
 Replication of Existing Objects
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-MinIO by default does not enable existing object replication. Objects
-created before replication was configured *or* while replication is
-disabled are not synchronized to the target deployment.
-MinIO supports enabling replication of existing objects in a bucket. 
-
-Enabling existing object replication marks all objects or object prefixes that
-satisfy the replication rules as eligible for synchronization to the source
-cluster, *even if* those objects were created prior to configuring or enabling
-replication. You can enable existing object replication while configuring
-or modifying a replication rule:
-
-- For new replication rules, include ``"existing-objects"`` to the list of
-  replication features specified to :mc-cmd:`mc replicate add --replicate`.
-
-- For existing replication rules, add ``"existing-objects"`` to the list of
-  existing replication features using 
-  :mc-cmd:`mc replicate update --replicate`. You must specify *all* desired
-  replication features when editing the replication rule. 
-
-Enabling existing object replication does not increase the priority of objects
-pending replication. MinIO uses the same core 
-:ref:`replication scanner and queue system <minio-replication-process>` for
-detecting and synchronizing objects regardless of the enabled replication
-feature. The time required to fully synchronize a bucket depends on a number of
-factors, including but not limited to the current cluster replication load,
-overall cluster load, and the size of the namespace (all objects in the bucket).
+MinIO by default replicates existing objects in the source bucket to the configured remote. 
+Existing object replication marks all objects or object prefixes that satisfy the replication rules as eligible for synchronization to the source cluster, *even if* those objects were created prior to configuring or enabling replication. 
```

Comment:
I keep thinking this should be two sentences, but not coming up with an alternative I like better. Something to contemplate? 🤔

Reply:
```suggestion
Default object replication marks all objects or object prefixes that satisfy the replication rule as eligible for synchronization to the target cluster.
This means that objects created prior to configuring or enabling bucket replication are marked for synchronization to the target cluster by default bucket replication configurations.
```

Reply:
Also, we say "source cluster" when I think we mean "target cluster".